### PR TITLE
add tweepy as a requirement

### PR DESCRIPTION
**Pull request by decentralgabe**
tweepy was not installed, it should be

### CODE REVIEW

**Comment by ericlavoie** [comment]
You need to adjust the docs, the docs says tweepy must be installed.

**Comment by decentralgabe** [comment]
yes the docs do say that, but no other deps are optional, why should this be? especially considering if someone is using docker, they'll have to manually edit the docker file. simpler to just include the dep

**Comment by ericlavoie** [comment]
It's not an objection, it's a reminder to make the changes in the docs, and include it with this PR. Remove the reference to install if you include it in requirement installation list.

**Comment by decentralgabe** [comment]
> It's not an objection, it's a reminder to make the changes in the docs, and include it with this PR. Remove the reference to install if you include it in requirement installation list.

my misunderstanding, let me update

File: docs/plugins.md [hunk 1]
```diff
@@ -133,19 +133,13 @@ This plugin allows payout notifications to be sent via a Twitter tweet. This plu
 Follow this outline to generate the API tokens and secrets for the plugin:
 
 1. Sign up as a developer at https://developer.twitter.com/
-2. Create a Twitter app
+2. Create a Twitter app. N.B. you will need to apply for [Elevated Access](https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-level).
```
Comment (ericlavoie) [inline docs/plugins.md:136]: Good point, this is indeed something I needed to figure out when I set it up.

**Review by ericlavoie — APPROVED** [review]
Looks good to me, ask someone else to also review it.

**Comment by decentralgabe** [comment]
@ericlavoie I seem to be unable to add other reviewers, may be some permissions on the repo? 🤔

**Comment by ericlavoie** [comment]
Under review you do not have the request review option, or on top right?

**Comment by decentralgabe** [comment]
nope 😄

**Review by jdsika — APPROVED** [review]

**Pull request merged**